docs/src/main/sphinx/release/release-415.md

```diff
@@ -0,0 +1,36 @@
+# Release 415 (25 Apr 2023)
+
+## General
+
+* Improve performance consistency of aggregations with variable file sizes. ({issue}`11361`)
```

Comment:
It's that performance is more stable / less variable? But not necessarily better?

Reply:
What do file sizes have anything to do with this, though? cc @sopel39

Reply:
I think its so that performance is equally good for smaller and larger files and any mix of them. Specifically this probably also means that performance does not vary as much if some of the data is compacted in object stores and partitions or so are not.